lib/hafriedlander/Peg/Parser/Basic.php

```diff
@@ -156,13 +156,13 @@ public function store(&$result, $subres, $storetag = \null) {
 		$storecalled = \false;
 
 		foreach ($this->typestack($result['_matchrule']) as $type) {
-			if ($method = $this->isCallable($storetag ? "{$type}_{$storetag}" : "{$type}_{$subres['name']}")) {
+			if ($this->isCallable($method = $storetag ? "{$type}_{$storetag}" : "{$type}_{$subres['name']}")) {
```

Comment:
Is there any practical reason for this change, too?

Reply:
isCallable() returns boolean value, there are not callable ;)